overview.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+In this tutorial, we'll run through the full story of a claim.
+
+To do so, three actors will be involved: a <span class="label-role claimer">claimer</span>, an <span class="label-role attester">attester</span> and a <span class="label-role verifier">verifier</span>. You'll be playing all three roles (unless you're in a KILT workshop and different participants are playing different roles).
+
+These three actors will be exchanging various objects; the most important one is the `attestedClaim`, aka credential.
+This is how an `attestedClaim` is created:
+
+<div class="mermaid">
```

Comment:
This does not work, it comes up incorrectly. 

![Screenshot 2020-02-24 at 14 04 26](https://user-images.githubusercontent.com/48058984/75154726-ab1a2f00-570e-11ea-8cf4-14b0ff438da1.png)

Reply:
Thanks - can you please pull and try again?

Reply:
Should be fixed now!

Reply:
Nice